index.html

```diff
@@ -57,14 +58,299 @@ <h3 class="activity">I am a Full Stack Developer</h2>
             <img src="./images/victory-icon.png" class="icon victory" alt="victory icon">
           </a>
         </li>
+        <li>
+          <a href="#">
+            <img src="./images/twitter-icon.png" class="icon twitter" alt="twitter icon">
+          </a>
+        </li>
         <li>
           <a href="#">
             <img src="./images/m-icon.png" class="icon gmail" alt="gmail icon">
           </a>
         </li>
       </ul>
-      
     </div>
   </section>
+  
+  <section class="mid-section">
+    <div class="item mrw">
+      <p>My Recent Works</p>
+    </div>
+    <div class="mid-border">
+      <img src="./images/border.png" alt="border">
+    </div>
+    <div class="feature-image">
+      <div class="img1">
+        <img src="./images/placeholder-img.png" alt="img1">
+      </div>
+      <div class="title1">
+        <p class="mps">Multi-Post Stories</p>
+      </div>
+      <div class="sup-txt1">
+        <p class="description2">
+          A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+      </div>
+      <div class="wrapper1">
+        <div class="box1 css">css</div>
+        <div class="box1 html">html</div>
+        <div class="box1 bootstrap">bootstrap</div>
+        <div class="box1 ruby">Ruby</div>
```

Comment:
- [ ] While this portion is styled nicely and might work as it is, I suggest you use ul and li tags here to structure this category in the works section. This improves the semantics in your code. 
![image](https://user-images.githubusercontent.com/107795045/219815466-f0966258-6cbe-4c15-8e3d-ae98767e2c66.png)